apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SessionPoolManager.cs

```diff
@@ -107,7 +111,18 @@ private static SpannerSettings CreateSpannerSettingsWithVersionHeader()
         /// <param name="logger">The logger to use. May be null, in which case the default logger is used.</param>
         /// <returns>A <see cref="SessionPoolManager"/> with the given options.</returns>
         public static SessionPoolManager Create(SessionPoolOptions options, Logger logger = null) =>
-            new SessionPoolManager(options, logger ?? Logger.DefaultLogger, CreateClientAsync);        
+            new SessionPoolManager(options, DefaultSpannerSettings(), logger ?? Logger.DefaultLogger, CreateClientAsync);
+
+        /// <summary>
+        /// Creates a <see cref="SessionPoolManager"/> with the specified SpannerSettings and options.
+        /// </summary>
+        /// <param name="spannerSettings">The SpannerSettings to use. May be null, in which case the default settings are used.</param>
+        /// <param name="options">The options to use. Must not be null.</param>
+        /// <param name="logger">The logger to use. May be null, in which case the logger returned by spannerSettings.Logger will be used.
+        /// If that is also null, the default logger is used.</param>
+        /// <returns>A <see cref="SessionPoolManager"/> with the given options.</returns>
+        public static SessionPoolManager Create(SessionPoolOptions options, SpannerSettings spannerSettings, Logger logger = null) =>
```

Comment:
+1 to removing the logger, and I think it's not ambiguos even if someone was specifying `null` for the logger before. From now on the compiler will pick the one that has no optional parameters, which is what Jon mentioned that was weird, but has no practical effects, because both methods end up doing the same.

Actually, I just tried it:

```
public void Method(int a, object o = null) { }
public void Method(int a, string s) { }
...
Method(5, null); // This binds to the one with the string parameter.
```